### PR TITLE
Define approving process for changes to the Code of Conduct

### PR DESCRIPTION
**Pull request by christian-bromann**
In the [last CPC working group session](https://hackmd.io/XmcHdDDARLm_lSXMIj_J7Q) we have been working on some CoC issues and realised that we don't have a clear definition on the approval process for them. To have more clarity here we suggest to align the condition for merging changes to the CoC with changes to the governance of the CPC.

cc @jorydotcom @joesepi @SaraJo

### CODE REVIEW

**Review by joesepi — APPROVED** [review]
LGTM. Thanks!

**Comment by rginn** [comment]
+1 Thanks!

**Review by ljharb — APPROVED** [review]

**Review by jorydotcom — APPROVED** [review]

**Review by mhdawson — APPROVED** [review]
LGTM

**Review by eemeli — CHANGES_REQUESTED** [review]
If we're expanding the scope of what's covered by the "governance of the CPC", we should make it really clear what is in fact covered. I would like this PR to include an explicit list of links to documents that is considered within the scope of that phrase. This should also make later changes easier to apply.

On the general idea of including the CoC to be a part of the "governance of the CPC", I definitely agree.

**Review by tobie — APPROVED** [review]

**Review by designMoreWeb — APPROVED** [review]
+1 great looks good to me

**Review by WaleedAshraf — APPROVED** [review]

File: governance/GOVERNANCE.md [hunk 1]
```diff
@@ -240,6 +240,8 @@ CPC members may request fast-tracking of pull requests they did not author. In t
...
 [Active OpenJS Collaborator]: #definition-of-an-active-openjs-collaborator
 [OpenJS Foundation CPC directory]: https://github.com/openjs-foundation/directory-private/blob/HEAD/groups/cross-project-council.yml
-[Primary CPC Director]: https://github.com/openjs-foundation/cross-project-council/blob/HEAD/CPC-CHARTER.md#the-primary-cpc-director-as-defined-in-43d-in-the-openjs-foundation-bylaws
-[Secondary CPC Director]: https://github.com/openjs-foundation/cross-project-council/blob/HEAD/CPC-CHARTER.md#the-secondary-cpc-director-as-defined-in-43e-in-theopenjs-foundation-bylaws
-[Tertiary CPC Director]: https://github.com/openjs-foundation/cross-project-council/blob/HEAD/CPC-CHARTER.md#the-tertiary-cpc-director-as-defined-in-43f-in-the-openjs-foundation-bylaws
+[Primary CPC Director]: ../CPC-CHARTER.md#the-primary-cpc-director-as-defined-in-43d-in-the-openjs-foundation-bylaws
+[Secondary CPC Director]: ../CPC-CHARTER.md#the-secondary-cpc-director-as-defined-in-43e-in-theopenjs-foundation-bylaws
+[Tertiary CPC Director]: ../CPC-CHARTER.md#the-tertiary-cpc-director-as-defined-in-43f-in-the-openjs-foundation-bylaws
+[Code of Conduct]: ../conduct
+[governance of the CPC]: ./
```
[the remark this answers is not in the document]
Comment (eemeli) [reply governance/GOVERNANCE.md:247]: @christian-bromann Have you had a chance to look at this?

**Comment by mikesamuel** [comment]
The remaining issue seems to be defining an appropriate link for

    [governance of the CPC]: ./

used in this context

> Pull requests that do not change the charter or **governance of the CPC** can be merged into
this repository provided the following conditions have been met:

as [pointed out](https://github.com/openjs-foundation/cross-project-council/pull/889#discussion_r878019912) by @eemeli.

Since the affected file is `governance/GOVERNANCE.md` it seems like the context is apparent without a link.
Could we do away with the link and get the rest committed?

**Comment by ovflowd** [comment]
Just a friendly bump to see if the requested changes can be addressed?

File: governance/GOVERNANCE.md (hunk 1 above)
[the remark this answers is not in the document]
Comment (eemeli) [reply governance/GOVERNANCE.md:247]: @christian-bromann Ping?

**Comment by tobie** [comment]
Obsoleted by https://github.com/openjs-foundation/cross-project-council/pull/1135 (the CPC is responsible for CoC changes).

**Pull request closed without merging**